Makefile

```diff
@@ -0,0 +1,21 @@
+IMAGE_REGISTRY ?= quay.io/openstack-k8s-operators
+IMAGE_TAG ?= latest
+
+# MUST_GATHER_IMAGE needs to be passed explicitly to avoid accidentally pushing to openstack-k8s-operators/must-gather
+ifndef MUST_GATHER_IMAGE
+$(error MUST_GATHER_IMAGE is not set.)
+endif
+
```

Comment:
I think it would be good to have a help target similar to what we have in install_yamls: https://github.com/openstack-k8s-operators/install_yamls/blob/main/Makefile#L356

Reply:
+1, makes sense, I'll create it in the next iteration.